graphql_compiler/query_pagination/__init__.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+from graphql.language.printer import print_ast
+
+from graphql_compiler.cost_estimation.cardinality_estimator import estimate_number_of_pages
+from graphql_compiler.query_pagination.parameter_generator import (
+    hydrate_parameters_of_parameterized_query
+)
+from graphql_compiler.query_pagination.query_parameterizer import generate_parameterized_queries
+
+
+PaginationQueries = namedtuple(
+    'PaginationQueries',
+    (
+        'next_page_query',          # Document, AST for the query that along with
```

Comment:
Is it more intuitive to call it `next` or `first`? I see how `next` relates to the python generator vocabulary, but in context here, I think `first and continuation` is a clearer partition of all results than `next and continuation`.

Reply:
How do you generate the second page query? By calling this function twice. At that point `first_page_query` should actually be called `second_page_query`, unless the user somehow knows that that `first` in `first_page_query` is relative to.

Reply:
Call it beginning then.

The problem is that `next` and `continuation` literally mean the same thing. Which one is nexter than the other?

Reply:
How about `next_page_query` and `remainder_query`?